gcampus/tools/views.py

```diff
@@ -0,0 +1,60 @@
+import math
+
+from django.utils.translation import gettext_lazy as _
+from django.views.generic import ListView, DetailView, TemplateView
+
+from gcampus.core.models import ParameterType
+from gcampus.tools.models import Calibration, MeasurementKit
+from gcampus.core.views.base import TitleMixin
+
+
+class ToolsOverView(TitleMixin, TemplateView):
+    title = _("Tools")
+    template_name = "gcampustools/tools.html"
+
+
+class MeasurementKitOverView(TitleMixin, ListView):
+    title = _("Measurement Kits")
+    template_name = "gcampustools/measurement_kit_overview.html"
+    model = MeasurementKit
+
+    def get_context_data(self, **kwargs):
+        available_kits = MeasurementKit.objects.filter(
+            calibrations__isnull=False
+        ).distinct()
+        kwargs["kits"] = available_kits
+        kwargs["row_num"] = math.ceil(len(available_kits) / 3)
+        kwargs["kits_num"] = len(available_kits)
+        return super(MeasurementKitOverView, self).get_context_data(**kwargs)
+
+
+class ODConverterOverView(TitleMixin, ListView):
+    title = _("Optical Density Converter")
+    template_name = "gcampustools/od_converter_overview.html"
+    model = ParameterType
+
+    def get_context_data(self, **kwargs):
+        available_parameters = ParameterType.objects.filter(
```

Comment:
Instead of this, you should probably overwrite the `get_queryset` method to make use of the features of the `ListView` class.